.github/workflows/build.yml

```diff
@@ -34,6 +34,12 @@ defaults:
 env:
   BUILD_ID: ${{ inputs.buildId }}
   BASHBREW_ARCH: ${{ inputs.bashbrewArch }}
+  BASHBREW_IMAGE_SIGNER: sha256:c7df9727f33c9b4b63ffe3214a2628860fc160f9d1ea1d601445058a053088d9
+  BASHBREW_REFERRERS_REPO: oisupport/referrers
+  BASHBREW_TUF_META: docker/tuf-metadata:latest
+  BASHBREW_TUF_TARGETS: docker/tuf-targets
```

Comment:
This is another thing that I think would be good to at least have a blurb about -- what this is, what it represents, how it's managed, what else it might be used for, etc.